### DISCUSSION TITLE
posts/db/5.ClickHouse/4.%E5%88%9D%E8%AF%86ClickHouse

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#429] [Announcements]
# posts/db/5.ClickHouse/4.%E5%88%9D%E8%AF%86ClickHouse

ClickHouse是一个用于联机分析(OLAP)的列式数据库管理系统(DBMS)。
什么是OLTP; 
全称 OnLine Transaction Processing，联机事务处理系统, 就是对数据的增删改查等操作; 
存储的是业务数据，来记录某类业务事件的发生，比如下单、支付、注册、等等; 
典型代表有Mysql、 Oracle等数据库，对应的网...

http://localhost:8080/posts/db/5.ClickHouse/4.%E5%88%9D%E8%AF%86ClickHouse.html

**Comment by xw-zzz** [comment]
11
